jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
   comment:
-    - add: 
+    - add:
         body: |~
           {{ or .overrides.comment "" | indent 10 }}
+{{end}}
+{{ if .overrides.label }}
```

Comment:
Yeah, thats a good Idea, What happens if you leave the -add: line empty there?

Is the program supposed to ignore it if its empty?  I noticed it does not do that for the comment section, but maybe it is supposed to?